zeekpkg/manager.py

```diff
@@ -2289,7 +2292,18 @@ def _stage(self, package, version, clone,
             return invalid_reason
 
         raw_metadata = _get_package_metadata(raw_metadata_parser)
-        requested_user_vars = user_vars(raw_metadata)
+
+        # Ensure any listed executables exist as advertised. Do this first so
+        # that we don't build anything if it fails.
+        for p in self._get_executables(raw_metadata):
+            full_path = os.path.join(clone.working_dir, p)
+            if not os.path.isfile(full_path):
+                return str.format("executable '{}' is missing", p)
+
+            if not os.access(full_path, os.X_OK):
+                return str.format("file '{}' is not executable", p)
```

Comment:
```suggestion
```
Guessing this sneaked in while resolving a Git conflict?  (This logic got moved to the end and shouldn't be here anymore).

Reply:
Whoops! Yes, thanks for catching this. I've committed your suggestion.